app/code/Magento/Integration/Model/CustomerTokenService.php

```diff
@@ -83,6 +92,7 @@ public function createCustomerAccessToken($username, $password)
                 __('You did not sign in correctly or your account is temporarily disabled.')
             );
         }
+        $this->eventManager->dispatch('customer_login', ['customer' => $customerDataObject]);
```

Comment:
Should this event be thrown in case when token used instead of when it is acquired?

Reply:
@paliarush Actually such an event should be thrown once a customer is logged id that is the case

Reply:
@slavvka  Just to confirm I was on right track or should I use a @paliarush suggestion or not?

Reply:
@prakashpatel07 after discussing this, I think current implementation is good. Thanks